pkg/mountkexec/mountkexec.go

```diff
@@ -41,14 +42,19 @@ func MountISO(isoPath, mountDir string) error {
 // KexecISO boots up a new kernel and initramfs
 func KexecISO(opp *webboot.Distro, path string) error {
 	var image boot.OSImage
+	kernelPath := opp.Kernel
+	if !filepath.IsAbs(kernelPath) {
+		kernelPath = filepath.Join(path, "/", opp.Kernel)
```

Comment:
you do not need the "/". filepath.Join will put it in for you.

Since you are going to change this anyway, could you change the argument name from 'path' to 'dir'? It's a bit confusing for me.

Reply:
Done.